src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -43,5 +43,10 @@ public override bool Verify(DataCache snapshot, Transaction tx)
             // on-chain transaction.
             return !NativeContract.Ledger.ContainsTransaction(snapshot, Hash);
         }
+
+        public override long CalculateNetworkFee(DataCache snapshot, Transaction tx)
```

Comment:
We also need to modify stte dependant transaction verification in `Transaction.cs` to ensure that network fee is enough to pay for all attributes as it was in the prewious PR.

Reply:
Done

Reply:
Resolved.